sdk/servicebus/service-bus/test/internal/unit/linkentity.unittest.spec.ts

```diff
@@ -283,6 +269,31 @@ describe("LinkEntity unit tests", () => {
       );
     });
 
+    it("abortSignal - is passed through to negotiateClaim", async () => {
+      const abortController = new AbortController();
+
+      let sawAbortSignal = false;
+      try {
+        const old = linkEntity["_negotiateClaim"];
+        linkEntity["_negotiateClaim"] = async (props) => {
+          if (props.abortSignal) {
+            sawAbortSignal = true;
+          }
+          abortController.abort();
+          return old.call(linkEntity, props);
+        };
+
+        await linkEntity.initLink({}, abortController.signal);
+        assert.fail("Should have thrown");
+      } catch (err) {
+        assert.isTrue(sawAbortSignal, "Should have seen the abortSignal.");
+        assert.deepNestedInclude(err, {
```

Comment:
Love this call!

Reply:
Thanks! Someone very wise suggested it to me once :smile: